test/system/mentor/categories_test.rb

```diff
@@ -94,4 +94,13 @@ class Mentor::CategoriesTest < ApplicationSystemTestCase
     find("a[href='/mentor/categories/#{category.id}/practices']").click
     assert_text '学習の準備カテゴリーのプラクティス並び替え'
   end
+
+  test 'should_display_practice_count_correctly' do
```

Comment:
```suggestion
  test 'should display practice count correctly' do
```

他に合わせて`_`ではなくスペースを使っていいと思います〜

Reply:
テスト名の`_`をスペースに修正しました。ご指摘ありがとうございます！